action.yml

```diff
@@ -11,18 +11,29 @@ inputs:
     required: true
   install-mock-db-objects:
     description: Specifies whether mock db objects should be used to fill out dependencies. If set to true mock-db-object-nuget-feed-url must also be set, otherwise an error will occur.
```

Comment:
We might want to call out specifically that you're expecting a `true` or `false` string value.